packages/application-shell/src/components/application-shell-authenticated/new-main-navigation-flag-wrapper.tsx

```diff
@@ -14,10 +15,9 @@ const NewMainNavigationFlagWrapper = (props: {
   }: TNewMainNavigationFlagWrapperChildrenProps) => JSX.Element;
 }) => {
   const isNewNavigationEnabled = useFeatureToggle(MAIN_NAVIGATION);
-  // TODO: A temporary workaround for a glitch in computing the aggregated adapter status within useAdapterStatus
-  const isReady =
-    // @ts-ignore
-    window?.__flopflip__?.launchdarkly?.getIsConfigurationStatus?.(2);
+  const { isReady } = useAdapterStatus({
+    adapterIdentifiers: [adapterIdentifiers.launchdarkly],
```

Comment:
Out of curiosity, what's the difference between `adapterIdentifiers.launchdarkly` and `adapterIdentifiers.http` ([reference](https://github.com/tdeekens/flopflip/blob/cb0928d237503cfc95d4954cb96e996c8cace286/packages/types/src/index.ts#L132))?

I would expect `http` to represent any adapter that requires HTTP calls to fetch its state. 🤔